Minor issue in BenchmarkCon example

There is an eyesore in the output of BenchmarkCon example -- the printing of the PID is not correct. Tested with V2 branch as currently it does not work under the master branch.

```
Copyright (c) 2010 Travis Robinson. <libusbdotnet@gmail.com>
website: http://sourceforge.net/projects/libusbdotnet
1. 1C7A:0575 Egis Technology Inc. - EgisTec Touch Fingerprint Sensor
2. 04D8:FA2E Microchip Technology, Inc. - Benchmark Device            (Note -- this line is good).

Select device (1-2) :2

Benchmark device 04D8:FFFFFA2E opened..              (Note-- this line is not good)
Loop Test Information
        Vid / Pid       : 04D8h / FFFFFA2Eh                      (Note -- this line is not good)
        Interface #     : 00h
        Alt Interface # : 00h
        Priority        : Normal
        Buffer Size     : 4096
        Buffer Count    : 1
        Display Refresh : 1000 (ms)
        Transfer Timeout: 5000 (ms)
        Retry Count     : 0
        Verify Data     : Off

Bulk Read (Ep81h) max packet size: 32
Bulk Write (Ep01h) max packet size: 32

While the test is running:
Press 'Q' to quit
Press 'T' for test details
Press 'I' for status information
Press 'R' to reset averages

Press 'Q' to exit, any other key to begin..

```

There are also quite some warnings when building.
```
mcuee@UbuntuSwift3:~/build/libusb/libusbdotnet_v2$ dotnet build ./stage/BenchmarkCon/BenchmarkCon.csproj -c Release
MSBuild version 17.5.1+f6fdcf537 for .NET
  Determining projects to restore...
  Restored /home/mcuee/build/libusb/libusbdotnet_v2/stage/BenchmarkCon/BenchmarkCon.csproj (in 183 ms).
  Restored /home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj (in 2.22 sec).
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(89,62): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(99,62): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(119,70): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/SetupApi.cs(239,77): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/SetupApi.cs(296,81): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/WinUsb/Internal/LibusbKAPI.cs(102,78): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/WinUsb/Internal/WinUsbAPI.cs(104,78): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/MonoLibUsb/MonoUsbEventHandler.cs(188,25): warning SYSLIB0006: 'Thread.Abort()' is obsolete: 'Thread.Abort is not supported and throws PlatformNotSupportedException.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsb/Internal/LibUsbRequest.cs(71,30): warning CS0618: 'Marshal.ReadByte(object, int)' is obsolete: 'ReadByte(Object, Int32) may be unavailable in future releases.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Main/UsbDeviceFinder.cs(261,20): warning SYSLIB0011: 'BinaryFormatter.Deserialize(Stream)' is obsolete: 'BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Main/UsbDeviceFinder.cs(272,13): warning SYSLIB0011: 'BinaryFormatter.Serialize(Stream, object)' is obsolete: 'BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
  LibUsbDotNet -> /home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/bin/Release/net6.0/LibUsbDotNet.LibUsbDotNet.dll
  BenchmarkCon -> /home/mcuee/build/libusb/libusbdotnet_v2/stage/BenchmarkCon/bin/Release/net6.0/BenchmarkCon.dll

Build succeeded.

/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(89,62): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(99,62): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/Kernel32.cs(119,70): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/SetupApi.cs(239,77): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Internal/SetupApi.cs(296,81): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/WinUsb/Internal/LibusbKAPI.cs(102,78): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/WinUsb/Internal/WinUsbAPI.cs(104,78): warning CS0618: 'UnmanagedType.AsAny' is obsolete: 'Marshalling arbitrary types may be unavailable in future releases. Specify the type you wish to marshal as.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/MonoLibUsb/MonoUsbEventHandler.cs(188,25): warning SYSLIB0006: 'Thread.Abort()' is obsolete: 'Thread.Abort is not supported and throws PlatformNotSupportedException.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsb/Internal/LibUsbRequest.cs(71,30): warning CS0618: 'Marshal.ReadByte(object, int)' is obsolete: 'ReadByte(Object, Int32) may be unavailable in future releases.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Main/UsbDeviceFinder.cs(261,20): warning SYSLIB0011: 'BinaryFormatter.Deserialize(Stream)' is obsolete: 'BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/Main/UsbDeviceFinder.cs(272,13): warning SYSLIB0011: 'BinaryFormatter.Serialize(Stream, object)' is obsolete: 'BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.' [/home/mcuee/build/libusb/libusbdotnet_v2/stage/LibUsbDotNet/LibUsbDotNet.csproj::TargetFramework=net6.0]
    11 Warning(s)
    0 Error(s)

Time Elapsed 00:00:05.62
```

It is running okay though, tested under Ubuntu Linux 20.04.

```
mcuee@UbuntuSwift3:~/build/libusb/libusbdotnet_v2$ ./stage/BenchmarkCon/bin/Release/net6.0/BenchmarkCon list
LibUsbDotNet USB Benchmark v2.2.0.0
Copyright (c) 2010 Travis Robinson. <libusbdotnet@gmail.com>
website: http://sourceforge.net/projects/libusbdotnet
1. 1D6B:0003 
2. 1C7A:0575 EgisTec EH575
3. 04F2:B6DD 
4. 046D:C52B 
5. 04D8:FA2E Travis Robinson - Benchmark Device
6. 05E3:0610 
7. 8087:0026 
8. 1D6B:0002 
9. 05E3:0612 
10. 1D6B:0003 
11. 1D6B:0002 

Select device (1-11) :5

Benchmark device 04D8:FFFFFA2E opened..
Loop Test Information
	Vid / Pid       : 04D8h / FFFFFA2Eh
	Interface #     : 00h
	Alt Interface # : 00h
	Priority        : Normal
	Buffer Size     : 4096
	Buffer Count    : 1
	Display Refresh : 1000 (ms)
	Transfer Timeout: 5000 (ms)
	Retry Count     : 0
	Verify Data     : Off

Bulk Read (Ep81h) max packet size: 32
Bulk Write (Ep01h) max packet size: 32

While the test is running:
Press 'Q' to quit
Press 'T' for test details
Press 'I' for status information
Press 'R' to reset averages

Press 'Q' to exit, any other key to begin..
Avg. Bytes/s: 281089.13 Transfers: 64 Bytes/s: 281089.13
Avg. Bytes/s: 281586.12 Transfers: 133 Bytes/s: 282048.68
Avg. Bytes/s: 281850.28 Transfers: 202 Bytes/s: 282360.87
Avg. Bytes/s: 281827.39 Transfers: 271 Bytes/s: 281760.38
Avg. Bytes/s: 281768.85 Transfers: 340 Bytes/s: 281539.17
Avg. Bytes/s: 281694.68 Transfers: 408 Bytes/s: 281324.42
Avg. Bytes/s: 281377.62 Transfers: 477 Bytes/s: 279517.33
Avg. Bytes/s: 281343.64 Transfers: 545 Bytes/s: 281105.51
Avg. Bytes/s: 281338.11 Transfers: 614 Bytes/s: 281294.46
Avg. Bytes/s: 281331.90 Transfers: 683 Bytes/s: 281276.63
Avg. Bytes/s: 281330.71 Transfers: 751 Bytes/s: 281318.77
Avg. Bytes/s: 281346.00 Transfers: 820 Bytes/s: 281512.48
qAvg. Bytes/s: 281347.22 Transfers: 889 Bytes/s: 281361.70
stopped Ep81h thread.
stopped Ep01h thread.
Loop Test Information
	Vid / Pid       : 04D8h / FFFFFA2Eh
	Interface #     : 00h
	Alt Interface # : 00h
	Priority        : Normal
	Buffer Size     : 4096
	Buffer Count    : 1
	Display Refresh : 1000 (ms)
	Transfer Timeout: 5000 (ms)
	Retry Count     : 0
	Verify Data     : Off

Bulk Read (Ep81h) max packet size: 32
	Total Bytes     : 3645440
	Total Transfers : 890
	Avg. Bytes/sec  : 281351.14
	Elapsed Time    : 12.96 seconds

Bulk Write (Ep01h) max packet size: 32
	Total Bytes     : 3645440
	Total Transfers : 890
	Avg. Bytes/sec  : 281352.76
	Elapsed Time    : 12.96 seconds

Press any key to exit..
```

--- a/stage/BenchmarkCon/BenchmarkConsole.cs
+++ b/stage/BenchmarkCon/BenchmarkConsole.cs
@@ -1261,7 +1261,7 @@ RefetchAltInterface:
                 }
             }
 
-            CONMSG("Benchmark device {0:X4}:{1:X4} opened..\n", Test.Vid, Test.Pid);
+            CONMSG("Benchmark device {0:X4}:{1:X4} opened..\n", ((ushort)Test.Vid), ((ushort)Test.Pid));
 
             // If this is a "whole" usb device (libusb-win32, linux libusb-1.0)
             // it exposes an IUsbDevice interface. If not (WinUSB) the 

